apps/web/src/pages/Pools/ViewPoolPage/components/UnArchiveProcessDialog.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import ArchiveBoxIcon from "@heroicons/react/24/solid/ArchiveBoxIcon";
+
+import { Dialog, Button } from "@gc-digital-talent/ui";
+
+import { ProcessDialogProps } from "./types";
+
+type UnarchiveProcessDialogProps = ProcessDialogProps & {
+  onUnarchive: () => Promise<void>;
+};
+
+const UnarchiveProcessDialog = ({
+  poolName,
+  isFetching,
+  onUnarchive,
+}: UnarchiveProcessDialogProps) => {
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const intl = useIntl();
+
+  const title = intl.formatMessage({
+    defaultMessage: "Un-archive process",
```

Comment:
Does "un-archiving" mean to restore?

Reply:
Sort of, but not quite since "restore" implies it was deleted. I just copy and pasted this from the existing dialog but now that you highlight it I think using the hyphen is incorrect. I believe the proper spelling is "unarchive".

Reply:
The term does not appear in dictionaries, so I am not sure what the "correct" spelling is. Will leave this since it already appears with the hyphen elsewhere.